bootstrap.yml

```diff
@@ -1301,10 +1301,21 @@ packages:
 
       
 tasks:
+  - name: make-useful-image
```

Comment:
Maybe we should call this `useful-image-dependencies` since it doesn't actually "make" anything.